examples/terraform/aws/output.tf

```diff
@@ -22,6 +22,10 @@ output "kubeone_api" {
   }
 }
 
+output "ssh_commands" {
+  value = formatlist("ssh -J ${var.bastion_user}@${aws_instance.bastion.public_ip} ${var.ssh_username}@%s", aws_instance.control_plane.*.private_ip)
+}
```

Comment:
Frankly, even this one made it here only because I've forgot to remove it. But as I got tired constantly constructing this line manually I made this.
It's whatever, but I don't want to touch other terraform configs as it will trigger way more jobs then I'm willing to wait for.,